config/base.yml

```diff
@@ -363,7 +363,7 @@ Lint/DeprecatedClassMethods:
 Lint/DeprecatedOpenSSLConstant:
   Enabled: true
 
-# Disabled because it can promote code that is overly DRY at the expense of readability -- see #TODO
+# Disabled because it can promote code that is overly DRY at the expense of readability -- see https://github.com/testdouble/standard/pull/228
```

Comment:
For the record we just delete things from the YAML file when we remove this (I'll fix this in master). Reason being that there are so many rules out there and having to visually parse enabled/disabled stuff or commented rules is just too much work. If someone wants to know the sordid history of a particular cop they can just `git log -S DuplicateBranch` and find this commit!